command/ibft/switch/ibft_switch.go

```diff
@@ -64,6 +64,14 @@ func setFlags(cmd *cobra.Command) {
 		"",
 		"the maximum number of validators in the validator set for PoS",
 	)
+
+	cmd.Flags().StringVar(
+		&params.customContractAddressRaw,
+		customContractAddress,
+		"",
+		"the custom contract [SC] address to use with PoA fork for reporting",
```

Comment:
Since this custom contract will be used for more than just reporting and more for just PoA, I think it would be better if we have this say something like this instead:

```
the custom contract address to use for SC interaction
```